dependencies.py

```diff
@@ -40,7 +40,7 @@
     Dependency("pyqtgraph", "pyqtgraph", ">=0.10.0"),
     Dependency("threedigrid", "threedigrid", "==1.0.13"),
     Dependency("cached-property", "cached_property", ""),
-    Dependency("threedi-modelchecker", "threedi_modelchecker", ""),
+    Dependency("threedi-modelchecker", "threedi_modelchecker", "==0.2"),
```

Comment:
Zitten we 1:1 aan deze specifieke versie vast of is 0.2.1 ook goed?
Als je een development git versie hiervan installeert om wat te testen krijg je denk ik een foutmelding omdat "0.3.dev0" niet de goede versie is.
```suggestion
    Dependency("threedi-modelchecker", "threedi_modelchecker", ">=0.2"),
```

Reply:
Hmm, dat is wat lastig. threedi-modelchecker zit momenteel wel vast aan een specifieke versie (migratie) van het 3di model. Threeditoolbox in het algemeen niet. Maar de integratie van threedi-modelchecker in ThreediToolbox dus wel.

Ik denk dat we hem daarom wel beter kunnen pinnen aan een specifieke versie. Anders kan de volgende situatie voorkomen:
1. Gebruiker A installeert ThreediToolbox met daarbij threedi-modelchecker 0.1.
2. threedi-modelchecker wordt upgedate (en gereleased) naar 0.2 om een nieuwe 3di migratie te ondersteuen/nieuwe checks implementeren.
3. Gebruiker B installeert dezelfde ThreediToolbox als gebruiker A maar krijgt nu threedi-modelchecker 0.2.

Nu is het gedrag van ThreediToolbox verschillend terwijl ze wel dezelfde versie van ThreediToolbox hebben geinstalleerd. Dit kan verwarrend zijn en lastige problemen opleveren (mijnes doet het wel maar jouwes niet!).